.changeset/angry-pots-yawn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/starlight': minor
```

Comment:
One quick query: any particular reason you think this should be a minor rather than patch release? In my head it felt like a safe patch fix, but do you think maybe people were depending specifically on the previous values here?

Reply:
Whoops, no, I meant to select that as a patch fix. Feel free to update—a little preoccupied for the rest of the day.

Reply:
Sure thing, I can do that — thanks again Eva! 💖